src/udisksdaemonutil.c

```diff
@@ -56,6 +56,22 @@
  * Various utility routines.
  */
 
+
+/**
+ * udisks_string_wipe_and_free:
+ * @string: A string with potentially unsafe content or %NULL.
+ *
+ * Wipes the buffer and frees the string.
+ */
+void udisks_string_wipe_and_free (GString *string)
+{
+  if (string != NULL)
+    {
+      memset (string->str, '\0', string->len);
+      g_string_free (string, TRUE);
+    }
+}
+
```

Comment:
It is already part of the first commit that uses it.

github shows these commits in a different order than they are in the git history. This is due to the fact, that commit order or time has changed at some point during a rebase. It confused me too, so I assume your comment may be based on the same misunderstanding. Or did I overlook something?

Everything should be `bisect` able fine. I think.